src/test/java/teammates/test/cases/ui/browsertests/FeedbackMcqQuestionUiTest.java

```diff
@@ -137,9 +137,9 @@ public void testCustomizeOptions() {
 
         ______TS("MCQ: add mcq option after remove");
 
-        feedbackEditPage.clickAddMoreMcqOptionLink();
+        feedbackEditPage.clickAddMoreMcqOptionLinkForNewQuestion();
         assertTrue(feedbackEditPage.isElementPresent("mcqOptionRow-3--1"));
-        feedbackEditPage.clickAddMoreMcqOptionLink();
+        feedbackEditPage.clickAddMoreMcqOptionLinkForNewQuestion();
```

Comment:
Not sure why a new question is handled in this manner here, when `FeedbackMsqQuestionUiTest.java` passes in `-1` as a parameter instead.